SIPS/sip-32.md

```diff
@@ -0,0 +1,89 @@
+---
+sip: 32
+title: Add `snap_trackEvent` method for pre-installed Snaps
+status: Draft
+author: Daniel Rocha (@danroc)
+created: 2025-03-20
+---
+
+## Abstract
+
+This Snap Improvement Proposal (SIP) introduces a new method,
+`snap_trackEvent`, allowing pre-installed Snaps to submit tracking events
+through the client. The client will determine how to handle these events.
+
+This feature enables Snaps to utilize the existing MetaMask client
+infrastructure for event tracking while maintaining user privacy.
+
+## Motivation
+
+Currently, there is no standardized way for pre-installed Snaps to submit
+analytics or tracking events. This proposal aims to:
+
+- Enable pre-installed Snaps to reuse MetaMask’s event tracking infrastructure.
```

Comment:
Enable pre-installed Snaps to _leverage_ MetaMask's event tracking infrastructure.